### ISSUE TITLE
zotero 7 beta.63 插件不兼容

### ISSUE THREAD

**Issue by yonghongwu** [MuiseDestiny/zotero-reference#238]
很多插件都不兼容，请问有什么解决办法嘛？还是说只能等后续修复了？

<img width="665" alt="image" src="https://github.com/MuiseDestiny/zotero-reference/assets/46832782/d8a405a2-0fda-4cb4-86e8-a7c333fede6c">

**Comment by ykny12058** [comment]
不报错但是无法正常使用

**Comment by FeralFlora** [comment]
> 不报错但是无法正常使用

Yeah, same here in Beta 64. Can't open the preference tab of Crush reference.

**Comment by FeralFlora** [comment]
> 很多插件都不兼容，请问有什么解决办法嘛？还是说只能等后续修复了？
> <img alt="image" width="665" src="https://private-user-images.githubusercontent.com/46832782/307582998-d8a405a2-0fda-4cb4-86e8-a7c333fede6c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkzNzcwMTYsIm5iZiI6MTcwOTM3NjcxNiwicGF0aCI6Ii80NjgzMjc4Mi8zMDc1ODI5OTgtZDhhNDA1YTItMGZkYS00Y2I0LTg2ZTgtYTdjMzMzZmVkZTZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzAyVDEwNTE1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3NWYyMmJkOWNkMjljMzFkYmQxNGY2ZmNkNDk4YWE4MjMzODMxNmE4ZDFmZTljMjRlMjMzYjI4YWIwNzc2YjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qIwbsINiE2_fCvt38wwchaNNMRQdILZ3dC4RpqCHuz0">

That's Better Bibtex, not Reference for Zotero.

**Comment by ykny12058** [comment]
> > 不报错但是无法正常使用
> 
> Yeah, same here in Beta 64. Can't open the preference tab of Crush reference.

Can I understand that the plugin is not available due to the update of the beta version of zotero, so I have to wait for the developer to update
